build/pmm-server-docker/Dockerfile

```diff
@@ -5,15 +5,16 @@ EXPOSE 80 443
 WORKDIR /opt
 
 RUN useradd -s /bin/false pmm
-RUN yum -y install epel-release && yum -y install ansible
+RUN yum -y install epel-release \
+    && yum -y install ansible
 
 COPY RPMS /tmp/RPMS
 COPY gitCommit /tmp/gitCommit
 
 COPY ansible /opt/ansible
 RUN cp -r /opt/ansible/roles /opt/ansible/pmm2-docker/roles
-RUN ansible-playbook -vvv -i 'localhost,' -c local /opt/ansible/pmm2-docker/main.yml \
-    && ansible-playbook -vvv -i 'localhost,' -c local /usr/share/pmm-update/ansible/playbook/tasks/update.yml \
+RUN ansible-playbook -vvv -i 'localhost,' -c local -e 'pmm2_server_repo=laboratory' /opt/ansible/pmm2-docker/main.yml \
+    && ansible-playbook -vvv -i 'localhost,' -c local -e 'pmm2_server_repo=laboratory' /usr/share/pmm-update/ansible/playbook/tasks/update.yml \
```

Comment:
How that will affect main branch autobuilds?

Reply:
As I said, these changes are temporary and they won't be used in upstream. Just an override for playbook vairable in scope of this FB.